app/workflows/start_rapidpro_flow.rb

```diff
@@ -0,0 +1,55 @@
+class StartRapidproFlow
+  attr_accessor :phone_call, :rapidpro_client
+
+  def initialize(phone_call, options = {})
+    self.phone_call = phone_call
+    self.rapidpro_client = options.fetch(:rapidpro_client) {
+      Rapidpro::Client.new(api_token: api_token)
+    }
+  end
+
+  def call
+    return if api_token.blank?
```

Comment:
Because `api_token` is required to make `Rapidpro::Client`, so better to raise ArgumentError in their constructor instead manually check here.

Reply:
Here I want to turn off the workflow if they don't set their API token.